wpilibc/src/main/native/include/frc/PWM.h

```diff
@@ -83,6 +84,7 @@ class PWM : public wpi::Sendable, public wpi::SendableHelper<PWM> {
    * Write a raw value to a PWM channel.
    *
    * @param value Raw PWM value.
```

Comment:
We should document this value as being pulse width in microseconds- it can be mentioned in the class level documentation as well, but having it here would be ideal.

Reply:
It's not exactly the pulse width in microseconds. Each step is a microsecond though. I couldn't figure out an easy way to explain it.

Reply:
Actually, on the roboRIO, it is milliseconds. You can have values over 2000. Thats actually how the nidec controller works. Its set to FFFF, which causes the output to always be high when the robot is enabled.